src/components/network-map-tab.js

```diff
@@ -712,7 +723,7 @@ export const NetworkMapTab = ({
             mapEquipments={mapEquipments}
             updatedLines={updatedLines}
             geoData={geoData}
-            waitingLoadGeoData={waitingLoadGeoData}
+            displayOverlayLoader={!basicDataReady && waitingLoadData}
```

Comment:
displayOverlayLoader={!basicDataReady}
it's enough nope ?

Reply:
I changed it after the comment of @jonenst here https://github.com/gridsuite/gridstudy-app/pull/1032#discussion_r1081579427
But actually if there an error all the screen get black with just the error message so yeah in practice it's enough I think.

Reply:
But maybe one day only the map screen will display the error ? I don't know.

Reply:
Ok to keep it

Reply:
yes it's good practice to never display a spinning loader "by default" (!data). Only display a loader when we know that there is an actual http request that we are waiting for (waiting && !data). Otherwise after the code is changed a few times and there are errors, the users may see infinite spinners because we're not actually waiting for anything